tests/integration_tests/model_tests.py

```diff
@@ -504,7 +504,7 @@ def test_sql_mutator(self):
         sql = tbl.get_query_str(query_obj)
         self.assertNotIn("-- COMMENT", sql)
 
-        def mutator(*args):
+        def mutator(*args, **kwargs):
```

Comment:
Can we write a better test for making sure the mutator properly takes the params?

Reply:
@hughhhh I added one more test to check that the database name is being passed in. LMK if you think more tests around that would be helpful. `user_name` was null and the `security_manager` is a function so printing them wouldn't really validate much.